parser/tests/sample_resource_json_payloads.bal

```diff
@@ -910,3 +910,73 @@ string TEST_FHIR_RESOURCE_XML_PATIENT_01_STRING = string `<Patient xmlns="http:/
 </Patient>`;
 
 string TEST_FHIR_RESOURCE_UNSUPPORTED_STRING = "hello";
+
+json TEST_FHIR_RESOURCE_JSON_INVALID_PATIENT_02 = {
```

Comment:
shall we add a test to check cardinality constraints as well?

Reply:
Added: https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/de58a7d86639b75fea6f884d0295c03b5aa462f9